tests/MessagePackAnalyzer.Tests/MessagePackAnalyzerTests.cs

```diff
@@ -1,23 +1,25 @@
 ﻿// Copyright (c) All contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Collections.Immutable;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Testing;
 using Microsoft.CodeAnalysis.Testing;
 using Xunit;
 using VerifyCS =
     CSharpCodeFixVerifier<MessagePackAnalyzer.MessagePackAnalyzer, MessagePackAnalyzer.MessagePackCodeFixProvider>;
 
 public class MessagePackAnalyzerTests
 {
-    private const string Preamble = @"
+    private const string Preamble = /* lang=c#-test */ @"
```

Comment:
Why did you add this code comment here and many other places?

Reply:
This enables syntax highlighting for the test code. 
See [Meziantou`s blog about this](https://www.meziantou.net/how-to-test-a-roslyn-analyzer.htm#get-syntax-highlight)

I could remove this (it’s just a personal preference).

Reply:
Oh! Interesting. Does that work in VS? Does it require an extension?

Reply:
I tried it, and yes it works in vanilla VS. Awesome. I knew about the attribute syntax but not the comment syntax. Let's keep what you have.